[4.2] インストールコマンド実行時にPHP Deprecatedが発生する

## 概要(Overview)

php8.1で確認しています。
インストールコマンド実行時、PHP Deprecatedが発生します(非対話モード)。

```
bin/console e:i -n

 Run doctrine:database:create...
 Created database var/eccube.db for connection named default

 Run doctrine:schema:drop --force...

                                                                                                                        
 [ERROR] The command "'bin/console' 'doctrine:schema:drop' '--force'" failed.                                           
                                                                                                                        
         Exit Code: 1(General error)                                                                                    
                                                                                                                        
         Working directory: /var/www/html/ec-cube/ec-cube                                                               
                                                                                                                        
         Output:                                                                                                        
         ================                                                                                               
                                                                                                                        
          Dropping database schema...                                                                                   
                                                                                                                        
                                                                                                                        
                                                                                                                        
         Error Output:                                                                                                  
         ================                                                                                               
         PHP Deprecated:  Return type of Eccube\Common\EccubeConfig::offsetExists($offset) should either be compatible with
         ArrayAccess::offsetExists(mixed $offset): bool, or the #[\ReturnTypeWillChange] attribute should be used to temporarily
         suppress the notice in /var/www/html/ec-cube/ec-cube/src/Eccube/Common/EccubeConfig.php on line 66             
         PHP Deprecated:  Return type of Eccube\Common\EccubeConfig::offsetGet($offset) should either be compatible with
         ArrayAccess::offsetGet(mixed $offset): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily
         suppress the notice in /var/www/html/ec-cube/ec-cube/src/Eccube/Common/EccubeConfig.php on line 76             
         PHP Deprecated:  Return type of Eccube\Common\EccubeConfig::offsetSet($offset, $value) should either be compatible with
         ArrayAccess::offsetSet(mixed $offset, mixed $value): void, or the #[\ReturnTypeWillChange] attribute should be used to
         temporarily suppress the notice in /var/www/html/ec-cube/ec-cube/src/Eccube/Common/EccubeConfig.php on line 85 
         PHP Deprecated:  Return type of Eccube\Common\EccubeConfig::offsetUnset($offset) should either be compatible with
         ArrayAccess::offsetUnset(mixed $offset): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily
         suppress the notice in /var/www/html/ec-cube/ec-cube/src/Eccube/Common/EccubeConfig.php on line 95             
         08:17:04 CRITICAL  [console] Error thrown while running command "doctrine:schema:drop --force". Message: "An exception
         occurred while executing a query: SQLSTATE[HY000]: General error: 1 near "(": syntax error" ["exception" =>    
         Doctrine\DBAL\Exception\SyntaxErrorException { …},"command" => "doctrine:schema:drop --force","message" => "An 
         exception occurred while executing a query: SQLSTATE[HY000]: General error: 1 near "(": syntax error"] ["file" =>
         "/var/www/html/ec-cube/ec-cube/vendor/symfony/console/EventListener/ErrorListener.php","line" => 48,"class" => 
         "Symfony\Component\Console\EventListener\ErrorListener","callType" => "->","function" => "onConsoleError","uid" =>
         "d6d3667","user_id" => "N/A","session_id" => "N/A"]                                                            
                                                                                                                        
         In ExceptionConverter.php line 71:                                                                             
                                                                                                                        
           An exception occurred while executing a query: SQLSTATE[HY000]: General err                                  
           or: 1 near "(": syntax error                                                                                 
                                                                                                                        
                                                                                                                        
         In Exception.php line 30:                                                                                      
                                                                                                                        
           SQLSTATE[HY000]: General error: 1 near "(": syntax error                                                     
                                                                                                                        
                                                                                                                        
         In Connection.php line 60:                                                                                     
                                                                                                                        
           SQLSTATE[HY000]: General error: 1 near "(": syntax error                                                     
                                                                                                                        
                                                                                                                        
         doctrine:schema:drop [--em EM] [--dump-sql] [-f|--force] [--full-database]  
```
## 期待する内容(Expect)  or 要望 (Requirement)


## 再現手順(Procedure)

```
$ tar xvzf eccube-4.2.0-beta.tar.gz 
$ cd ec-cube
$ export ... // 環境変数でDB情報等を定義
$ bin/console eccube:install -n
```

### 環境 (environment)
+ EC-CUBE: 4.2.0-beta
+ PHP: 7.x.x
+ DB:
  - PostgreSQL x.x.x 
  - MySQL x.x.x


## 関連情報 (Ref)

--- a/src/Eccube/Common/EccubeConfig.php
+++ b/src/Eccube/Common/EccubeConfig.php
@@ -50,12 +50,10 @@ class EccubeConfig implements \ArrayAccess
     /**
      * @param $key
      * @param $value
-     *
-     * @return mixed
      */
     public function set($key, $value)
     {
-        return $this->container->setParameter($key, $value);
+        $this->container->setParameter($key, $value);
     }
 
     /**
@@ -63,6 +61,7 @@ class EccubeConfig implements \ArrayAccess
      *
      * @return bool
      */
+    #[\ReturnTypeWillChange]
     public function offsetExists($offset)
     {
         return $this->has($offset);
@@ -73,6 +72,7 @@ class EccubeConfig implements \ArrayAccess
      *
      * @return mixed
      */
+    #[\ReturnTypeWillChange]
     public function offsetGet($offset)
     {
         return $this->get($offset);
@@ -82,6 +82,7 @@ class EccubeConfig implements \ArrayAccess
      * @param mixed $offset
      * @param mixed $value
      */
+    #[\ReturnTypeWillChange]
     public function offsetSet($offset, $value)
     {
         $this->set($offset, $value);
@@ -92,6 +93,7 @@ class EccubeConfig implements \ArrayAccess
      *
      * @throws \Exception
      */
+    #[\ReturnTypeWillChange]
     public function offsetUnset($offset)
     {
         throw new \Exception();

--- a/src/Eccube/Entity/AbstractEntity.php
+++ b/src/Eccube/Entity/AbstractEntity.php
@@ -28,6 +28,7 @@ use Symfony\Component\Serializer\Serializer;
 /** @MappedSuperclass */
 abstract class AbstractEntity implements \ArrayAccess
 {
+    #[\ReturnTypeWillChange]
     public function offsetExists($offset)
     {
         $inflector = new Inflector(new NoopWordInflector(), new NoopWordInflector());
@@ -39,10 +40,12 @@ abstract class AbstractEntity implements \ArrayAccess
             || method_exists($this, "has$method");
     }
 
+    #[\ReturnTypeWillChange]
     public function offsetSet($offset, $value)
     {
     }
 
+    #[\ReturnTypeWillChange]
     public function offsetGet($offset)
     {
         $inflector = new Inflector(new NoopWordInflector(), new NoopWordInflector());
@@ -59,6 +62,7 @@ abstract class AbstractEntity implements \ArrayAccess
         }
     }
 
+    #[\ReturnTypeWillChange]
     public function offsetUnset($offset)
     {
     }

